### PR TITLE
about.md: Include the presence of gVisor as additional security layer.

### PR DESCRIPTION
**Pull request by EtiennePerot**
This is a continuation of #41, rebased on `main` and including all of @apyrgio's comments.

### CODE REVIEW

File: src/about.md [hunk 1]
```diff
@@ -48,23 +48,23 @@ I got the idea for Dangerzone from Qubes, an operating system that runs everythi
...
 
-When Dangerzone starts the container that will sanitize the suspicious document, it _disables networking_ and does not mount anything. So if a malicious document hacks the container, it doesn’t have access to your data and it can’t use the internet, so there’s not much it could do.
+When Dangerzone starts the container that will sanitize the suspicious document, it will first start a gVisor sandbox _inside_ that container, then run the potentially-dangerous document processing workload inside the sandbox. This ensures that the process dealing with the document is isolated from the Linux kernel. The sandbox and its parent container are also both configured to _disable networking_ and to not mount anything from the host filesystem. So if a malicious document manages to execute arbitrary code, this code doesn’t have access to the host kernel, doesn't have access to your data, and can't use the internet, so there’s not much it could do.
 
-Here’s how it works. The first container:
+Here’s how it works. The first sandbox:
```
Comment (almet) [inline src/about.md:60]: In this version of the document, we're first referencing "the first sandbox", and then "the second container". Maybe changing the phrasing to say what happens first and what happens next can streamline the read?

```suggestion
Here’s how it works. First, the sandbox:
```

**Review by almet — COMMENTED** [review]
Thanks for proposing these changes Etienne, they explain well how gVisor helps security-wise.

(I've added a small comment on phrasing)

File: src/about.md (hunk 1 above)
Comment (apyrgio) [reply src/about.md:60]: Right, there's no first and second sandbox, per our distinction above. At the same time, I feel that the term "the sandbox" is a bit too vague. Maybe a more accurate description is:

```suggestion
Here’s how it works. The first containers starts a gVisor sandbox *inside* it which performs the following actions:
```

**Review by apyrgio — APPROVED** [review]

**Comment by apyrgio** [comment]
Aside from a nit that Alexis mentioned, we can merge this branch.

File: src/about.md (hunk 1 above)
Comment (almet) [reply src/about.md:60]: It feel adding this last change would be a repeat, b/c what we call "the sandbox" is defined just in the previous paragraph. I understand your fear of being "too vague", but I don't think it's the case here.

Comment (EtiennePerot) [reply src/about.md]: I will incorporate this now, sorry for the delay.

Comment (EtiennePerot) [reply src/about.md]: Done.

**Comment by almet** [comment]
Thanks @EtiennePerot (and for the final squash).

This is now live at https://dangerzone.rocks/about/

**Pull request merged**